code/game/machinery/computer/buildandrepair.dm

```diff
@@ -117,6 +117,10 @@
 	name = "Circuit board (Security Records)"
 	desc = "A circuit board for running a computer used for viewing security records."
 	build_path = /obj/machinery/computer/secure_data
+/obj/item/weapon/circuitboard/security_alerts
```

Comment:
as far as I can tell you never actually added a way to get more of these
it's in a weird spot because if you make it something science can print, they will have their own personal security CMC to combo with their telescience, yet if you don't make it printable, someone can break in, unscrew the circuitboard, and weld it, then there's no way to ever get a new computer

You could add it to the ammolathe I guess, but that's also weird since it's not ammo.